server/build_event_protocol/build_status_reporter/build_status_reporter.go

```diff
@@ -198,13 +200,37 @@ func (r *BuildStatusReporter) githubPayloadFromTestSummaryEvent(event *build_eve
 
 func (r *BuildStatusReporter) githubPayloadFromFinishedEvent(event *build_event_stream.BuildEvent) *github.GithubStatusPayload {
 	description := descriptionFromExitCodeName(event.GetFinished().ExitCode.Name)
```

Comment:
Can we make this something a bit more user friendly than `OK in 32s` maybe something like `Succeeded in 32s` or `Passed in 32s` or `Success in 32s`

Reply:
Went with "Successful" which is what GH does for actions